digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAssetFiller.scala

```diff
@@ -3,8 +3,8 @@ package fi.liikennevirasto.digiroad2.linearasset
 import fi.liikennevirasto.digiroad2.asset.SideCode
 
 object LinearAssetFiller {
-  case class MValueAdjustment(assetId: Long, linkId: String, startMeasure: Double, endMeasure: Double)
-  case class VVHChangesAdjustment(assetId: Long, linkId: String, startMeasure: Double, endMeasure: Double, timeStamp: Long)
+  case class MValueAdjustment(assetId: Long, linkId: String, startMeasure: Double, endMeasure: Double,timeStamp: Long=0)
```

Comment:
Meillä ei taida olla enää tarvetta käyttää näitä adjustment timestamp, taitaa olla VVH samuutuksen jäänne. Pitäisi myös miettiä, kuinka toimitaan lrm_position, lane_position, lane_history_position ja road_link_attributes taulujen adjusted_timestamp kenttien kanssa. Näihin taidettiin aikaisemmin tallentaa VVH-samuutuksen aikaleima.

Reply:
jätin nyt ihan varmuuden vuoksi, varmaan vois ehkä weekly keskutella tän tarpeesta ja mihin käytettää.

Reply:
kysytty asia rocket chat, ei nyt tässä aleta korjaamaan asiaa